examples/pitch-field-demo/field_pitch.js

```diff
@@ -34,14 +34,14 @@ CustomFields.FieldPitch = function(text) {
 
   /**
    * Click event data.
-   * @type {?Blockly.EventData}
+   * @type {?Blockly.eventHandling.Data}
    * @private
    */
   this.clickWrapper_ = null;
 
   /**
    * Move event data.
-   * @type {?Blockly.EventData}
+   * @type {?Blockly.eventHandling.Data}
```

Comment:
Should this be `Blockly.browserEvents.Data`? I can't seem to find `Blockly.eventHandling` in core.